android/src/main/java/tech/bam/RNBraintreeDropIn/RNBraintreeDropInModule.java

```diff
@@ -47,6 +51,35 @@ public void show(final ReadableMap options, final Promise promise) {
 
     DropInRequest dropInRequest = new DropInRequest().clientToken(options.getString("clientToken"));
 
+    if (options.hasKey("googlePay")) {
+      final ReadableMap googlePayOptions = options.getMap("googlePay");
+      if (!googlePayOptions.hasKey("amount")) {
+        promise.reject("NO_GOOGLE_PAY_AMOUNT", "You must provide an amount for Google Pay");
+        return;
+      }
+      if (!googlePayOptions.hasKey("currencyCode")) {
+        promise.reject("NO_GOOGLE_PAY_CURRENCY_CODE", "You must provide an currency code for Google Pay");
+        return;
+      }
+
+      GooglePaymentRequest googlePaymentRequest = new GooglePaymentRequest()
+              .transactionInfo(TransactionInfo.newBuilder()
+                      .setTotalPrice(String.valueOf(googlePayOptions.getDouble("amount")))
+                      .setTotalPriceStatus(WalletConstants.TOTAL_PRICE_STATUS_FINAL)
+                      .setCurrencyCode(googlePayOptions.getString("currencyCode"))
+                      .build());
+      if (googlePayOptions.hasKey("merchantID")) {
+        googlePaymentRequest.googleMerchantId(googlePayOptions.getString("merchantID")); 
+      }
+      if (googlePayOptions.hasKey("merchantName")) {
+        googlePaymentRequest.googleMerchantName(googlePayOptions.getString("merchantName"));
+      }
+      googlePaymentRequest.googleMerchantName("Simi");
+      dropInRequest.googlePaymentRequest(googlePaymentRequest);
+    } else {
+      dropInRequest.disableGooglePayment();
```

Comment:
Is this line really necessary, and if yes does it work when you remove the gradle deps that you added earlier?

Reply:
This line is required to disable Google pay when developer doesn't pass "googlePay" parameter